### PR TITLE
Add kphuslog plugin

### PR DESCRIPTION
**Pull request by artemklevtsov**
Added simple plugin for the `github.com/phuslu/log`.

### CODE REVIEW

File: plugin/kphuslog/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+kphuslog
+========
+
+kphuslog is a plug-in package to hook [log](https://github.com/phuslu/log)
```
Comment (twmb) [inline plugin/kphuslog/README.md:4]: Can `[log]` be changed to `[phuslu/log]`?

**Review by twmb — COMMENTED** [review]
Minor comment, otherwise lgtm. Should this be 1.0 tagged or do you foresee API changes / breakages?

**Comment by twmb** [comment]
(and if you could, squash the commits)

**Comment by artemklevtsov** [comment]
How can I do it? If I right understand you can do it during [merge PR](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests).

**Comment by twmb** [comment]
I mean to squash your two commits into one with:

`git rebase -i master`
(in the new window, change the second `pick` to `s`, then exit)
(in the editor, keep the first commit message)

I can do it with a merge PR, but I don't like that it changes commit SHAs -- it makes writing changelogs a bit more tedious.

**Comment by artemklevtsov** [comment]
Seems done. I followed for the [SO answer](https://stackoverflow.com/a/25357146/1863950).

**Comment by twmb** [comment]
awesome ty, lgtm. Should this be released as as 1.0?

**Comment by artemklevtsov** [comment]
> Should this be released as as 1.0?
As you decide.

**Comment by twmb** [comment]
Added package documentation, lgtm.

**Comment by twmb** [comment]
Thanks! released as plugin/kphuslog/v1.0.0

**Pull request merged**